### PR TITLE
EVA-2214 RS Release page

### PR DESCRIPTION
**Pull request by andresfsilva**

### CODE REVIEW

File: src/js/rs-release/rs-release.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    this.id = Utils.genId("EvaRsRelease");
+    _.extend(this, args);
+    this.render();
+}
+
+EvaRsRelease.prototype = {
+    render: function() {
+        var releaseVersion = $.urlParam('releaseVersion');
+        if (!releaseVersion) {
+            this.updateUrl({releaseVersion: 2})
```
Comment (sundarvenkata-EBI) [inline src/js/rs-release/rs-release.js:31]: I am not a big fan of defaulting to a hardcoded release version... Could we make the API tolerant of absent releaseVersion parameter and return data for the most recent release version in such a scenario?

File: src/js/rs-release/rs-release.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+                                '<th><div title="RS IDs that should NOT be used because they are merged into another active RS">Merged RS</div></th>' +
+                                '<th><div title="RS IDs that should NOT be used since these RS IDs were deprecated">Deprecated RS</div></th>' +
+                                '<th><div title="RS IDs that should NOT be used because they have been merged into a deprecated RS">Merged Deprecated RS</div></th>' +
+                                '<th><div title="RS IDs that could NOT be mapped against an assembly">Unmapped RS</div></th>' +
+                            '</tr>' +
+                        '</thead>' +
+                        '<tbody>';
+
+        _.each(releaseData, function (species) {
+            var scientificNameNormalized = species.scientificName.toLowerCase().replace(' ', '_');
```
Comment (sundarvenkata-EBI) [inline src/js/rs-release/rs-release.js:105]: I suggest to replicate the functionality [here](https://github.com/EBIvariation/eva-accession/pull/246/files/23e335472055f8bfe29fa714d12250b0c9cae030#diff-b6d7f7cf30896264d9e49bd1ebe1bed7e80706b5ecf202be868fad3ad451a245R110) as inelegant as it is :(

**Review by sundarvenkata-EBI — CHANGES_REQUESTED** [review]
Looks good... Suggested change in constructing the scientific name...

File: src/js/rs-release/rs-release.js (hunk 1 above)
Comment (andresfsilva) [reply src/js/rs-release/rs-release.js]: I implemented an extra endpoint `/info/latest` to get the latest release and use it as the default if it is not specified in the url

File: src/js/rs-release/rs-release.js (hunk 2 above)
Comment (andresfsilva) [reply src/js/rs-release/rs-release.js]: The release folder was included in the database so the API is now retrieving that and constructing the links. The website is not constructing the links anymore.

**Review by sundarvenkata-EBI — APPROVED** [review]

**Review by sundarvenkata-EBI — APPROVED** [review]

**Review by tcezard — APPROVED** [review]

File: src/js/rs-release/rs-release.js [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+                } catch (e) {
+                    console.log(e);
+                }
+            }
+        });
+
+        var content = '<div><h2>Clustered variants (RS) Release</h2></div>' +
+                    '<div class="callout success">' +
+                        '<p>' +
+                            'The RS ID release (v.' + releaseVersion + ') is now available in our FTP. ' +
```
Comment (tcezard) [inline src/js/rs-release/rs-release.js:69]:
```suggestion
                            releaseInfo.releaseDate + ': The RS ID release (v.' + releaseVersion + ') is available in our FTP. ' +
```

**Review by sundarvenkata-EBI — APPROVED** [review]

**Pull request merged**